src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -531,14 +531,15 @@ object TezosTypes {
       level: Int,
       delegate: String,
       priority: Int,
-      estimated_time: java.sql.Timestamp
+      estimated_time: ZonedDateTime
```

Comment:
Curious, Why have we moved away from simple timestamps?

Reply:
I've suggested this to keep the domain model clean of things related to a database.
This helps immediately to decode json datetimes with no custom decoding definitions, and allows (if ever needed) more handle to operate on the datetime values.